judges/quality-of-service/quality-of-service.rb

```diff
@@ -32,15 +32,25 @@
   total = 0
   success = 0
   duration = 0
+  mttrs = []
+  mttr = {}
```

Comment:
Naming could be improved. It's not supposed to have multiple mttr. MTTR is an average value of multiple time values. Thus, the name should be smth like `repair_times` or `ttrs` (times to repair)